nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -952,7 +952,47 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(
 			// The object has no text, but we do want to render its children.
 			auto [varChildren, accChildRes] = getAccessibleChildren(pacc, 0, childCount);
 			if (S_OK != accChildRes || varChildren.size() == 0) {
-				LOG_ERROR(L"AccessibleChildren failed (count: " << childCount << L"), res: " << accChildRes);
+				std::wstringstream msg;
+				msg << L"AccessibleChildren failed (count: " << childCount << L"), res: " << accChildRes;
+				switch (accChildRes) {
+				case E_NOINTERFACE:
+					msg << L" (E_NOINTERFACE, No such interface supported)";
+					// This is certainly an error, and indicates a bug in the IA2 provider.
+					LOG_ERROR(msg.str());
+					break;
+				case RPC_E_DISCONNECTED:
+					msg << L" (RPC_E_DISCONNECTED, object invoked has disconnected from its clients.)";
+					// RPC_E_DISCONNECTED indicates that the parent died since the query
+					// to accChildCount, and returning no children indicates that all children

```

Comment:
I'd remove everything after "accChildCount" in this comment. There are no children to return if the parent died. The "no children" is the case you handle below.